src/integration-test/java/se/sundsvall/incidentmapper/apptest/IncidentsIT.java

```diff
@@ -44,10 +45,11 @@ void test02_updatedIssueFromPob() {
 		incidentRepository.saveAndFlush(IncidentEntity.create()
 			.withPobIssueKey("12345")
 			.withJiraIssueKey("UF-5974")
+			.withMunicipalityId("2281")
```

Comment:
Could make it a constant to only have to change it in once place.

Reply:
Fixed!